python_modules/libraries/dagster-dbt/dagster_dbt/asset_defs.py

```diff
@@ -639,7 +640,7 @@ def load_assets_from_dbt_project(
 
 
 def load_assets_from_dbt_manifest(
-    manifest: Optional[Mapping[str, Any]] = None,
+    manifest: Optional[Union[Path, Mapping[str, Any]]] = None,
```

Comment:
Use https://docs.python.org/3/library/os.html#os.PathLike so we can support string paths as well
```suggestion
    manifest: Optional[Union[os.PathLike, Mapping[str, Any]]] = None,
```